pkg/kubeapiserver/admission/initializer.go

```diff
@@ -67,6 +75,14 @@ func (i pluginInitializer) Initialize(plugin admission.Interface) {
 		wants.SetInternalClientSet(i.internalClient)
 	}
 
+	if wants, ok := plugin.(WantsToRun); ok {
```

Comment:
I don't mind to run one ``Run`` after all the admissions are initialized. E.g. ``admissionController.Run(stopCh)``. As  all the admissions are initialized at one place in cluster capacity framework. This would work for me as well.